src/qgitrebase.h

```diff
@@ -41,18 +41,16 @@ class LIBQGIT2_EXPORT Rebase
      * Create a wrapper for a libgit2 rebase object. Constructed object
      * takes ownership of \a rebase and frees it in the destructor.
      * @param rebase Raw pointer to the libgit2 rebase object
-     * @param opts Specifies how the rebase is to be conducted
      */
-    explicit Rebase(git_rebase *rebase, const RebaseOptions &opts);
+    explicit Rebase(git_rebase *rebase);
```

Comment:
The forward declaration on line 28 can be now removed.

Reply:
Removed.